solidity-frontend/src/main/java/org/sonarsource/solidity/frontend/Utils.java

```diff
@@ -8,4 +12,26 @@ private Utils() {
   public static String trimQuotes(String value) {
     return value.substring(1, value.length() - 1);
   }
+
+  // ... parser
+  public static SolidityParser returnParserUnitFromParsedFile(String file) {
+    CharStream cs = CharStreams.fromString(file);
+    SolidityLexer sl = new SolidityLexer(cs);
+    // TokenStream tokens = new CommonTokenStream(sl);
+    CommonTokenStream tokenStream = new CommonTokenStream(sl);
+    SolidityParser parser = new SolidityParser(tokenStream);
+    tokenStream.fill();
+    parser.handleComments(tokenStream.getTokens());
+    return parser;
+  }
+
+  public static SolidityParser returnParserFromParsedFile(CharStream cs) {
+    SolidityLexer sl = new SolidityLexer(cs);
+    // TokenStream tokens = new CommonTokenStream(sl);
```

Comment:
![Code Smell](https://sonarsource.github.io/sonar-github/code_smell.png 'Code Smell') Code Smell: This block of commented-out lines of code should be removed. ([squid:CommentedOutCodeLine](https://sonarcloud.io/organizations/stylianos-agapiou-sonarsource-gi/rules#rule_key=squid%3ACommentedOutCodeLine))

[See it in SonarCloud](https://sonarcloud.io/project/issues?id=org.sonarsource.solidity%3Asonar-solidity&pullRequest=14&issues=AWQCVB8NuoX1TGVkLMgm&open=AWQCVB8NuoX1TGVkLMgm)

[ISSUE_KEY]: AWQCVB8NuoX1TGVkLMgm